compiler-core/src/warning.rs

```diff
@@ -192,8 +254,8 @@ your program.",
         buffer
             .write_all(b"\n")
             .expect("error pretty buffer write space before");
-        let (diagnostic, extra) = self.to_diagnostic();
-        write_old(buffer, diagnostic, Severity::Warning);
+        let (_diagnostic, extra) = self.to_diagnostic();
+        // write_old(buffer, diagnostic, Severity::Warning);
```

Comment:
Could we put this code back please? So it's clear there is work still to do

Reply:
Sorry, I forgot to put this back. Done!